src/controllers/EthereumWalletController.ts

```diff
@@ -691,88 +875,6 @@ export class EthereumWalletController extends AbstractWalletController {
 		}
 	}
 
-	async signBulkMail(
```

Comment:
Remove - since signing will be done directly in SendMail. Client only needs to pass "deadline" parameter in options. Everything else is handled in WalletController and Wrappers.